recipes/pyobjc-framework-coreservices/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pyobjc-framework-CoreServices" %}
+{% set version = "8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 58bad844c8e5ee82852d860b117d1238c3ed2db8dd57f754feb6bd5801fe878d
+  patches:
+    - patches/0001-Remove-werror-from-compile-flags.patch
+
+build:
+  skip: true   # [not osx]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
```suggestion
```
I think the cxx compiler is not needed, and the link inspector is complaining about overdepending on the cxx standard library.